src/java/com/unifina/feed/util/StreamMessageComparator.java

```diff
@@ -0,0 +1,13 @@
+package com.unifina.feed.util;
+
+import com.streamr.client.protocol.message_layer.StreamMessage;
+
+import java.util.Comparator;
+
+public class StreamMessageComparator implements Comparator<StreamMessage> {
```

Comment:
This class could implement Serializable and to allow it to work with an ordered collection. Ordered collection can be serialized, but only if the comparator is also serializable. Comparator holds no state.